include/geolib/Box.h

```diff
@@ -14,13 +14,14 @@ class Box : public Shape {
     Box* clone() const;
 
     bool intersect(const Ray &, float t0, float t1, double& distance) const;
-
-    double getMaxRadius() const;
-
     bool intersect(const Box& other) const;
-
+    bool intersect(const Vector3& p, const double radius) const;
```

Comment:
I think we should add docstrings to all (new) function definitions. I'm not entirely sure what this method does. 

The fact that this was not done when this package was developed should not be a reason not to start now ;-)

Reply:
Done

Reply:
So what' s the difference between `intersect` and `contains` (with the same signature)?